index.html

```diff
@@ -0,0 +1,401 @@
+            class="link"
+            target="_blank"
+            rel="noreferrer noopener"
+          >
+            Get my resume
+          </a>
+          <img class="about-me-icon" src="./media/svg/icon-2.svg" alt="Icon" />
+        </section>
+        <hr />
+        <div class="about-me-section stacks">
+          <ul>
+            <li class="languages list-title">
+              Languages
+              <ul>
+                <li>JavaScript</li>
+                <li>TypeScript</li>
+                <li>Ruby</li>
+                <li>HTML</li>
+                <li>CSS</li>
+                <li>Sass</li>
+              </ul>
+            </li>
+          </ul>
+          <ul>
+            <li class="frameworks list-title">
+              Frameworks
+              <ul>
+                <li>React & Redux</li>
+                <li>Material-UI</li>
+                <li>Jest</li>
+                <li>Bootstrap</li>
+                <li>Ruby on Rails</li>
+                <li>RSPec</li>
+              </ul>
+            </li>
+          </ul>
+          <ul>
+            <li class="skills list-title">
+              Skills
+              <ul>
+                <li>GitHub</li>
+                <li>CodePen</li>
+                <li>GitLab</li>
+                <li>Terminal</li>
+                <li>WSL</li>
+                <li>Remote Pair-Programming</li>
+                <li>Teamwork</li>
+                <li>Mentoring</li>
+                <li>Code Reviews</li>
+                <li>Mobile/Responsive Development</li>
+              </ul>
+            </li>
+          </ul>
+        </div>
+      </div>
+      <section class="contact" id="contact">
+        <h2 class="intro">
+          I'm always interested in hearing about new projects, so if you'd like
+          to chat please get in touch.
+        </h2>
+        <form id="form" action="https://formspree.io/f/mrgjqpap" method="POST">
+          <label hidden>Name</label>
+          <input
+            class="form-fields"
+            type="text"
+            name="name"
+            id="name"
+            maxlength="30"
+            placeholder="Full Name"
+            required
+          />
+          <label hidden>Email Address</label>
+          <input
+            class="form-fields"
+            type="email"
+            name="Email Address"
+            id="email-address"
+            placeholder="Email Address"
+            required
+          />
+          <small id="email-invalidation-message"></small>
+          <label hidden>Subject</label>
+          <textarea class="form-fields" name="message" id="message" maxlength="500" required>Describe your subject here...</textarea>
+          <button
+            class="link"
+            id="get-in-touch"
+            type="submit"
+          >
+          Get in touch
+          </button>
+        </form>
+        <footer id="footer">
```

Comment:
Please consider giving this footer a margin to match the Figma design as requested by the previous reviewer.
Your design : 
![padding2](https://user-images.githubusercontent.com/95412943/211049484-5358970a-859a-430d-95df-ec208c36f08f.PNG)
 Figma design : 
![margin](https://user-images.githubusercontent.com/95412943/211049547-aa8db3c6-01a9-4092-b9be-b6928b505436.PNG)